### DISCUSSION TITLE
What About Gaussian process Regression?

### DISCUSSION THREAD

**Discussion by ranbir7** [Autostronomy/AstroPhot#58] [General]
Can u give me some suggestions in making some _code/approaches/ideas_ 
which will help me make some contributions towards it?

**Comment by ConnorStoneAstro** [comment]
Gaussian Processes can be used as an optimization algorithm that is very robust and fast. However it is complex to implement. There is already a package for doing Gaussian Processes with PyTorch, so it would require interfacing with that to optimize model parameters. Here is a link to the package: https://gpytorch.ai/

**Comment by ranbir7** [comment]
okay I'll be checking it out!
